PoGo.NecroBot.Logic/ILogicSettings.cs

```diff
@@ -96,11 +103,7 @@ public interface ILogicSettings
         int SnipeLocationServerPort { get; }
         bool UseSnipeLocationServer { get; }
         bool UseTransferIVForSnipe { get; }
-        bool SnipeIgnoreUnknownIV { get; }
         int MinDelayBetweenSnipes { get; }
-        int TotalAmountOfPokebalsToKeep { get; }
-        int TotalAmountOfPotionsToKeep { get; }
-        int TotalAmountOfRevivesToKeep { get; }
```

Comment:
I didn't, they got added to the master later than I pulled the master.  Am updating as we speak